documentation/public/user-guide/rust-crate/index.md

```diff
@@ -1,4 +1,8 @@
-# Cargo Crate
+# Rust Crate
+
```

Comment:
Thinking out loud: if we are going to have multiple `how_to-***.md` for each language, should we move the architecture/technical information to the common overview/architecture section instead, and leave these to be purely about practical language-specific code examples?

Reply:
Also, I wonder if we should even have content in `language/index.md`, or just convert all of it into the same HOW-TO structure:

- `how-to-install/index.md`
- `how-to-parse-a-file/index.md`
- etc...

Reply:
> should we move the architecture/technical information to the common overview/architecture section instead

That's what I think and did here; however, I think we should probably also have some language-specific intro (e.g. how to install the binary, maybe the feature flags), the API Overview or maybe some additional resources and have the how-to guides alongside them for now.

I tried to not over-structure it here and I imagine a better structure will naturally emerge in time, as we add more and more resources.